src/pages/Startpage.tsx

```diff
@@ -4,6 +4,8 @@ import Button from '@material-ui/core/Button';
 import Grid from '@material-ui/core/Grid';
 import { Link } from 'react-router-dom';
 import Typography from '@material-ui/core/Typography';
+import { Hidden } from '@material-ui/core';
+import DynamicTitle from '../components/DynamicTitle';
```

Comment:
Both of these lines need better imports.

```suggestion
import Hidden from '@material-ui/core/Hidden';
import DynamicTitle from 'components/DynamicTitle';
```